GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java

```diff
@@ -1,35 +1,33 @@
 package hackkerank.day1;
-import challenge.Icecream;
 import org.apache.log4j.Logger;
 
 import java.util.Scanner;
 
 public class IfElse {
 
-    public static Logger logJava = Logger.getLogger(IfElse.class);
-
-    private static final Scanner scanner = new Scanner(System.in);
+    public static Logger LOG_JAVA = Logger.getLogger(IfElse.class);
+    private static final Scanner SCANNER = new Scanner(System.in);
 
     public static void main(String[] args) {
-        int N = scanner.nextInt();
-        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+        int number = SCANNER.nextInt();
+        SCANNER.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
 
-        scanner.close();
+        SCANNER.close();
 
-        if (N%2==0 & 2<=N & N<=5){
-            logJava.info("Not Weird");
+        if (number%2==0 & 2<=number & number<=5){
```

Comment:
This is the only class that is not in the correct format, remember all binary operators except . should be separated from their operands by spaces. Here you can find more information https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html